libs/components/ag-grid/src/lib/modules/ag-grid/types/lookup-properties.ts

```diff
@@ -9,27 +9,106 @@ import {
 } from '@skyux/lookup';
 
 export interface SkyAgGridLookupProperties {
+  /**
+   * Fires when users select the button to add options to the list.
+   */
   addClick?: (args: SkyLookupAddClickEventArgs) => void;
+  /**
+   * The `aria-label` text for the lookup cell. If neither `ariaLabel` or `ariaLabelledBy` are specified, the `aria-label` defaults to the column's `headerName`, `headerTooltip`, `field`, or `colId`.
+   */
   ariaLabel?: string;
+  /**
+   * The HTML element ID of the element that labels the lookup cell. If neither `ariaLabel` or `ariaLabelledBy` are specified, the `aria-label` defaults to the column's `headerName`, `headerTooltip`, `field`, or `colId`.
+   */
   ariaLabelledBy?: string;
-  /** @deprecated */
+  /**
+   * The value for the autocomplete attribute on the form input.
+   * @default "off"
+   * @deprecated
+   */
   autocompleteAttribute?: string;
+  /**
+   * The data source for the lookup cell to search when users enter text. You can specify static data such as an array of objects, or you can pull data from a database.
+   */
   data?: unknown[];
+  /**
+   * How many milliseconds to wait before searching while users enter text in the lookup field.
+   * @default 0
+   * @deprecated Use the `searchAsync` event emitter and callback instead to provide data to the lookup component.
+   */
   debounceTime?: number;
+  /**
+   * The object property to display in the text input after users select an item in the dropdown list.
+   * @default "name"
+   */
   descriptorProperty?: string;
+  /**
+   * Whether to disable the lookup cell.
+   * @default false
+   */
   disabled?: boolean;
+  /**
+   * Whether to enable users to open a picker where they can view all options.
+   * @default false
+   */
   enableShowMore?: boolean;
+  /**
+   * The object property that represents the object's unique identifier. Specifying this property enables token animations and more efficient rendering. This property is required when using `enableShowMore` and `searchAsync` together.
```

Comment:
"object property ... object's" seems circular, so just checking to make sure this is correct.

Reply:
I think so--it is specifying which of the object's properties is a unique identifier for the object as a whole

Reply:
```suggestion
   * The property that represents the object's unique identifier. Specifying this property enables token animations and more efficient rendering. This property is required when using `enableShowMore` and `searchAsync` together.
```
I _think_ this edit removes the slight awkwardness and aligns with your explanation without changing the meaning.